README.md

```diff
@@ -1,57 +1,22 @@
-# Node.js Internationalization
+# Node.js i18n
 
 [![Crowdin](https://badges.crowdin.net/nodejs/localized.svg)](https://crowdin.com/project/nodejs)
 
+This repo is a basement of Node.js projects internationalization initiative. Here you can find contribution guides, discuss ideas or join meetings. Here also contains original and translated Node.js API documentation.
+
 ## Contributing
 
-Help us get the documentation translated into your native language and reach more people's hearts.
-**Visit [Crowdin/project/NodeJs](https://crowdin.com/project/nodejs)** and help translating docs
-
-### Languages being Translated
-
-- [Arabic](https://crowdin.com/project/nodejs/ar)
-- [Chinese Simplified](https://crowdin.com/project/nodejs/zh-CN)
-- [Chinese Traditional](https://crowdin.com/project/nodejs/zh-TW)
-- [Croatian](https://crowdin.com/project/nodejs/hr)
-- [Czech](https://crowdin.com/project/nodejs/cs)
-- [Danish](https://crowdin.com/project/nodejs/da)
-- [Dutch](https://crowdin.com/project/nodejs/nl)
-- [Finnish](https://crowdin.com/project/nodejs/fi)
-- [French](https://crowdin.com/project/nodejs/fr)
-- [German](https://crowdin.com/project/nodejs/de)
-- [Greek](https://crowdin.com/project/nodejs/el)
-- [Hebrew](https://crowdin.com/project/nodejs/he)
-- [Hindi](https://crowdin.com/project/nodejs/hi)
-- [Hungarian](https://crowdin.com/project/nodejs/hu)
-- [Indonesian](https://crowdin.com/project/nodejs/id)
-- [Italian](https://crowdin.com/project/nodejs/it)
-- [Japanese](https://crowdin.com/project/nodejs/ja)
-- [Korean](https://crowdin.com/project/nodejs/ko)
-- [Norwegian](https://crowdin.com/project/nodejs/no)
-- [Polish](https://crowdin.com/project/nodejs/pl)
-- [Portuguese](https://crowdin.com/project/nodejs/pt-PT)
-- [Portuguese (Brazil)](https://crowdin.com/project/nodejs/pt-BR)
-- [Romanian](https://crowdin.com/project/nodejs/ro)
-- [Russian](https://crowdin.com/project/nodejs/ru)
-- [Serbian (Latin)](https://crowdin.com/project/nodejs/sr-CS)
-- [Slovak](https://crowdin.com/project/nodejs/sk)
-- [Spanish](https://crowdin.com/project/nodejs/es-ES)
-- [Swedish](https://crowdin.com/project/nodejs/sv-SE)
-- [Thai](https://crowdin.com/project/nodejs/th)
-- [Turkish](https://crowdin.com/project/nodejs/tr)
-- [Ukrainian](https://crowdin.com/project/nodejs/uk)
-- [Vietnamese](https://crowdin.com/project/nodejs/vi)
-
-### CrowdIn Glossary
-
-We have a [CrowdIn Glossary](crowdin-glossary.json) that has some terms which are noted as shared or translatable across the project. If a term is missing, please add it! This glossary will automatically be used by CrowdIn to let translators know about each term when they come across them.
+Help us get the Node.js translated into your native language and reach more people's hearts. Please read next guides before:
```

Comment:
Either drop 'the':
> ~the~ Node.js
… or add project
> the Node.js *project*

Reply:
.. or `the Node.js project documentation`

Reply:
This text related not only to documentation, so I just dropped `the`